UserDocs.md

```diff
@@ -17,242 +17,228 @@ Install T4:
+* Copy the package contents to `/my/package/path` in your bucket according to their logical keys.
 
-### Snapshots and history
+Once the dataset has been materialized on T4, it will be available to anyone else with access to that T4 bucket.
 
-#### `helium.snapshot(path, message)`
-Creates a snapshot of the T4 object at `path` with commit message `message`.
 
-#### `helium.list_snapshots(bucket, contains=None)`
-List all snapshots in `bucket`. `contains=` is an optional parameter that limits
-the results to snapshots that contain the specified prefix.
+### Pulling a package from T4
+To pull a package that has been published to T4:
 
-Returns path, hash, timestamp, and snapshot message.
+	# load the package from the registry
+	p = t4.Package("nice-name", registry="name-of-your-t4-bucket")
 
-#### `helium.diff(bucket, src, dest)`
-Returns a list of differences between the two snapshots.
-Differences are one of three types: Add, Modify, or Delete.
+This will find the manifest associated with the given name and registry and downloads it into memory. This will not download the actual bytes. To do that, you can use `get` or `get_file`:
 
-`src` and `dest` are strings containing the snapshot's SHA-256 hash, or the string
-`"latest"`. `"latest"` signifies "what's currently in S3", but `"latest"` is not
-a proper snapshot.
+	# download everything from a package
+	p.copy("/", "target/directory/")
 
-`bucket` may not contain a terminating `/`.
+	# download a specific entry from a package
+	p.copy("foo.parquet", "target/directory/foo.parquet")
 
-`src` is the hash of the source snapshot, and `dest` is the hash of the destination snapshot.
-If `dest` contains file that `src` does not, that fill will appear as an Add.
-(If you were to swap the order of the parameters, the same file would appear as Delete.)
+	# load a specific entry into memory
+    data, meta = p.get("foo.parquet")
 
-### Configuration
+Especially with large data packages, there are many cases in which you might want to download a manifest describing a package, without downloading the actual contents of the package itself. This T4 API provides this functionality.
 
-#### `helium.config()`
-Returns the current configuration details as an `ordereddict` subclass, `HeliumConfig`.
-`HeliumConfig` has a cleaner appearance, and includes the file path.
+<!-- Note that in order for `get` to work, the file must have been created by t4.put and it must be in one of the formats supported by the default T4 deserializers. -->
 
-#### `helium.config(URL)`
-Set the local configuration from the URL of a T4 instance.  Overwrites all local config 
-values if successful, but keeps a backup in the config dir.
 
-#### `helium.config(KEY=VALUE [, KEY2=VALUE2, ...])`
-Manually sets specific configuration options.
+### Moving data not on T4
+We encourage a push/pull architecture based on having data packages ultimately live on T4. But you may also use the t4 CLI to handle moving data anywhere else you can path to.
```

Comment:
change "anywhere else you can path to" to something clearer. Maybe anywhere on your local file system. Or, to any location you can access...